docs/_sass/_main.scss

```diff
@@ -84,7 +93,6 @@ em {
 }
 
 a {
-    color: $color-light-blue;
```

Comment:
Why are we getting rid of the link color here? I would think this would just be `$color-link`